### DISCUSSION TITLE
Plugin Licenses

### DISCUSSION THREAD

**Discussion by lesbrunn** [apohl79/audiogridder#490] [Q&A] [answered]
Hi,

Is it necessary to have plugins installed on both computers or only on the server? If it's both, then one has to have multiple licenses per plugin?

**Comment by emack999** [comment, marked as answer]
Just the machine you're running the Audiogridder server application on.

**Reply by lesbrunn** [reply]
Thanks. What about AAX plugins? What would their location be?

**Comment by lesbrunn** [comment]
Hi guys,
I'm still not certain where AAX plugins should go. I don't think Pro Tools can access them if they are not in their default location. So how does AudioGridder handle AAX?

**Reply by apohl79** [reply]
AAX plugins can only be hosted by Pro Tools. If you have AAX only plugins with no VST/AU version, you have to install them on your DAW machine, as you can't run them through AG.

**Reply by lesbrunn** [reply]
Thanks. I misunderstood when I saw that the AG plugin was supported by AAX. I immediately went into overdrive and turned it upside down to mean that AG handles AAX... Now, that would be something!

**Reply by apohl79** [reply]
I see. Just for my understanding, for what would you need AG to host AAX plugins? Do you have plugins that are not available as VST/VST3 or AU but only AAX? The plugin support was needed to get AG to work out of Pro Tools. But I have not seen a vendor that offered a plugin only as AAX.

**Reply by lesbrunn** [reply]
Pro Tools handles CPU much less efficiently than other DAWs. As such, many DAWs work well with moderately powered CPUs, but even with the most powerful CPUs, Pro Tools still needs more, especially with VIs. It would be great to take the entire plugin load off Pro Tools and let it just handle audio, which it does excellently.

**Reply by lesbrunn** [reply]
Avid makes only AAX plugins

**Reply by apohl79** [reply]
As long as the plugins you use are available as AU/VST, you can do that with AG by loading the AG AAX and run VST/AU versions of your plugins on the AG server side.

**Reply by lesbrunn** [reply]
Would it be impossible to take Avid's AAX only plugins and give the load to AG?

**Reply by apohl79** [reply]
No, as I said, you can only access VST/AU through AG. So your plugins have to be available in non AAX formats.

**Reply by lesbrunn** [reply]
OK, got it. Even though I haven't yet used AG, I will when I have the time to set it up. I do use VE Pro and even though it works well, I see AG as a more hassle-free approach because you don't need multiple keyboards and mice. You are doing a tremendous job for free, and I will donate as soon as I set AG up and get it working. Would you consider writing a comprehensive manual to cover all aspects of the setup?

**Reply by apohl79** [reply]
Sounds good! I try to keep the documentation up to date on the website. But tbh, it is not easy to write a manual for non-developers, because I think the website covers all important aspects - but obviously it does not. So consider this being WIP. We will probably have at least a FAQ page at some point that should cover most common issues.

**Reply by lesbrunn** [reply]
Thanks again bro. More grease!
